worlds/AutoWorld.py

```diff
@@ -90,6 +90,8 @@ def call_all(world: "MultiWorld", method_name: str, *args: Any) -> None:
                         f"Duplicate item reference of \"{item.name}\" in \"{world.worlds[player].game}\" "
                         f"of player \"{world.player_name[player]}\". Please make a copy instead.")
 
+    # TODO: investigate: Iterating through a set is not a deterministic order.
+    # If any random is used, this could make unreproducible seed.
```

Comment:
oh, that's interesting. there might not be a problem currently, because it requires *two* worlds to use rng in the same stage, but it's definitely a bug.